deploy/main.tf

```diff
@@ -50,29 +52,48 @@ resource "google_project_service" "places" {
   disable_on_destroy = false
 }
 
-# Create a service account for cloud run
+
+
+#---------------------------------------------------------#
+# ------------------- Service Accounts -------------------#
+#---------------------------------------------------------#
+
+# Create a service account for the gpf-backend
 resource "google_service_account" "backend_sa" {
   account_id = "backend-sa"
   depends_on = [google_project_service.iam]
 }
 
+
+# Create a service account for the dlr-backend
+resource "google_service_account" "dlr_backend_sa" {
+  account_id = "dlr-backend-sa"
+  depends_on = [google_project_service.iam]
+}
+
 locals {
-  backend_serviceaccount = "serviceAccount:${google_service_account.backend_sa.email}"
+  backend_serviceaccount     = "serviceAccount:${google_service_account.backend_sa.email}"
+  dlr_backend_serviceaccount = "serviceAccount:${google_service_account.dlr_backend_sa.email}"
 }
 
-# Grant the service account access to databases and cloud run
+
+# Grant the service accounts access to databases and cloud run
 resource "google_project_iam_binding" "service_permissions" {
   project = var.project
   for_each = toset([
     "run.admin", "cloudsql.client"
   ])
 
   role    = "roles/${each.key}"
-  members = [local.backend_serviceaccount]
+  members = [local.backend_serviceaccount, local.dlr_backend_serviceaccount]
 }
 
+#---------------------------------------------------------#
+# ----------------------- DATABASE -----------------------#
+#---------------------------------------------------------#
+
 
-# Create the sql database for Venture Capitol
+# Create the main Postgres Instance
 resource "google_sql_database_instance" "vc_db" {
   name                = "vc-backend-database"
```

Comment:
Laufen unsere beiden Datenbanken (Also GPF und DLR) beide über die eine Instanz?
Wenn ja, dann sollte ja alles passen. Ansonsten sieht es so aus, als ob nur eine kreiert wird.

Reply:
Genau, eine Postgres instanz, in der alle Datenbanken, sowohl prod als auch staging laufen